### ISSUE TITLE
WSL: Separate volume for data

### ISSUE THREAD

**Issue by mook-as** [rancher-sandbox/rancher-desktop#584]
We may want to consider having a second volume for data on WSL (mounted into the main distro at `/var`, etc.), since we seem to be bumping the WSL distro more these days.  Currently, we request that people wipe out everything on some upgrades.

[ ] Figure out what paths we need to persist
[ ] Add a new distro for data
[ ] Figure out how to do shared mounts (via `/mnt/wsl/`)

**Comment by smerschjohann** [comment]
You can share mounts by simply bind mounting them into /mnt/wsl.

E.g.:

```
if [ ! -d /mnt/wsl/rancher-desktop-data ]; then
    mkdir /mnt/wsl/rancher-desktop-data
    wsl.exe -d rancher-desktop-data mount --bind /data /mnt/wsl/rancher-desktop-data/
fi
```

**Comment by mook-as** [comment]
This was implemented in #642 for Rancher Desktop 0.5.0 (September 2021).
